features/features.go

```diff
@@ -687,14 +639,6 @@ var (
 							enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 							mustRegister()
 
-	FeatureGateProcMountType = newFeatureGate("ProcMountType").
-					reportProblemsToJiraComponent("Node").
-					contactPerson("haircommander").
-					productScope(kubernetes).
-					enhancementPR("https://github.com/kubernetes/enhancements/issues/4265").
-					enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
-					mustRegister()
-
```

Comment:
Enabled by default in 1.33: https://github.com/kubernetes/kubernetes/blob/7cd7fea30d054efe4f2cc60073a29d2c618258ec/pkg/features/kube_features.go#L1619-L1623

@haircommander should we drop this?

Reply:
yes we should!

Reply:
Is it locked on upstream, or just on by default?

We may instead want to promote the feature to GA here and retain the ability to disable it if we find issues, for at least 1 more release